add fourth unique modifier to Vorana's March

hexeaktivitat opened the pull request:
Fixes #5617 & fixes #5444.

I also added explicit definitions for all 4 unique mods, though I don't really know which ones people are likely to choose.

QuickStick123 commented:
There is also https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/5444 which is similar if you wanted to tackle another issue.

hexeaktivitat commented:
Added a change to address #5444, and added "None" to the variant/mod list for both items. I feel like there's a better way to add it than just putting it in the array manually as I have, and there also doesn't seem to be a way to distinguish between patch versions for these generated items, since the variant system is being used for these mods themselves.

Regisle commented:
#5680 messes with variants a bit and might help with legacy variants for generated items, but its still WIP
